sunpy/map/mapbase.py

```diff
@@ -657,6 +657,20 @@ def date(self):
         This is taken from the 'DATE-OBS' FITS keyword.
         """
         time = self.meta.get('date-obs', None)
+        # Get the time scale
+        if time is not None and 'TAI' in time:
+            # SDO specifies the 'TAI' scale in their time string, which is parsed
```

Comment:
Stupid question, why was this not moved to the SDO map source? I can't find a comment or any reference.

Reply:
Why was what not moved to the map source?

Reply:
This code deals with SDO specific time handing no?

Reply:
Just in case there is other data out there that has this TAI timestamp, I think it's worth keeping it in here.